src/main/java/net/dv8tion/jda/api/entities/ThreadChannel.java

```diff
@@ -101,6 +102,31 @@ default ThreadMember getThreadMemberById(String id)
     @Nullable
     ThreadMember getThreadMemberById(long id);
 
+    @NotNull
```

Comment:
Have we decided on which annotations we want to use? It looks like we are still using javax

Reply:
We are going to continue using javax until further into v5 development.
There are a lot of conversations about which to use and frankly the ecosystem is still very fractured.

The answers to this question lay out the options and problems pretty well:
https://stackoverflow.com/questions/4963300/which-notnull-java-annotation-should-i-use

Reply:
(So this should be updated to use `@Nonnull` and `@Nullable` from `javax` for now)

Reply:
Has been replaced ^^